features/keychain/module/__tests__/keychain.integration-test.js

```diff
@@ -1,288 +0,0 @@
-import createInMemoryStorage from '@exodus/storage-memory'
```

Comment:
Can we keep the integration test? Or why was it removed?

Reply:
oops, died in rebase: aea74e7c8305c65b9ea24e4acf540d779d93dfca